src/components/Tabs/Tabs.scss

```diff
@@ -244,3 +304,11 @@ $focus-state-box-shadow-color: rgba(92, 106, 196, 0.8);
   visibility: hidden;
   height: 0;
 }
+
+// stylelint-disable selector-max-class
+.newDesignLanguage {
+  padding: $tab-separation-spacing spacing()
+    ($tab-separation-spacing - $border-bottom-height) spacing();
+  margin-left: $tab-separation-spacing;
+  margin-bottom: spacing(extra-tight);
```

Comment:
Note:

Can we de-globalize these?

Reply:
What do you mean? Not used the function?

Reply:
De-globalize in that these styles are put within a class such as `Tab` or `title` (haven't dug into which one specifically)